tattle-made/Uli
# [#544] Enable Slur Metadata Checklist Functionality

---
name: Integration Report
about: This PR is about integration of Enable Slur Metadata Functionality into the plugin
title: Enable Slur Metadata Checklist Functionality
labels: feature

---

**Describe the PR**
This PR is related to the issue #607.
Related comments are [this](https://github.com/tattle-made/Uli/issues/544#issuecomment-2186100849), [this](https://github.com/tattle-made/Uli/issues/607#issuecomment-2298217916), [this](https://github.com/tattle-made/Uli/issues/607#issuecomment-2305415519) and [this](https://github.com/tattle-made/Uli/issues/607#issuecomment-2314290506)

**Steps to test the PR**
It is similar to `Enable Slur Replacement` feature. Once the plugin is activated, you can click on `Elable Slur Metadata` to see metadata related to slur words. 
Detailed video is linked in this comment [here](https://github.com/tattle-made/Uli/issues/607#issuecomment-2314290506)
